File: ManagingLeads/ViewMostRecentLeads/ViewMostRecentLeads.cs
using System;
using RestSharp;

namespace ViewMostRecentLeads
{
    class ViewMostRecentLeads
    {
        static void Main(string[] args)
        {
            var client = new RestClient("https://api.apexchat.com/Services/ApexChatService.svc/leads/?start=0&limit=2");
            var request = new RestRequest(Method.GET);
            request.AddHeader("apexchat-company", "CompanyName");
            request.AddHeader("apexchat-username", "Username");
            request.AddHeader("apexchat-password", "Password");
            IRestResponse response = client.Execute(request);
            Console.WriteLine(response.Content);
        }
    }
}

File: ManagingLeads/ViewMostRecentLeads/ViewMostRecentLeads.java
package ManagingLeads.ViewMostRecentLeads;

import com.mashape.unirest.http.*;
import com.mashape.unirest.http.exceptions.UnirestException;

public class ViewMostRecentLeads {
    public static void main(String[] args) throws UnirestException {
        HttpResponse<String> response = Unirest
                .get("https://api.apexchat.com/Services/ApexChatService.svc/leads/?start=0&limit=2")
                .header("apexchat-company", "CompanyName")
                .header("apexchat-username", "Username")
                .header("apexchat-password", "Password")
                .asString();
        System.out.println(response.getBody());
    }
}